code/globalincs/scp_defines.h

```diff
@@ -6,4 +6,7 @@
 #include "scp_compiler_detection.h"
 #include "platformChecks.h"
 
+#define SCP_TOKEN_CONCAT1(x, y) x ## y
+#define SCP_TOKEN_CONCAT(x, y) SCP_TOKEN_CONCAT1(x, y)
```

Comment:
It's specifically for allowing to use `__LINE__` to be used with that macro. If only the first variant was used then "__LINE__" would appear in the source code as opposed to the line number in the file.